File: exercicio01/javascript/Exercicio05.js
function getMaiorEMenorElemento(array) {
    const retorno = {
        maior: array[0],
        menor: array[0]
    };

    array.forEach(elemento => {
        if(elemento > retorno.maior) {
            retorno.maior = elemento;
        }

        if(elemento < retorno.menor) {
            retorno.menor = elemento;
        }
    });

    return retorno;
}

File: exercicio01/java/src/Exercicio05.java
public class Exercicio05 {
    public static void mostraMaiorEMenorValor(int[] array) {
        int maior = array[0];
        int menor = array[0];

        for (int elemento : array) {
            if(elemento > maior) {
                maior = elemento;
            }

            if(elemento < menor) {
                menor = elemento;
            }
        }

        System.out.println("Maior: " + maior);
        System.out.println("Menor: " + menor);
    }
}
